### ISSUE TITLE
Error while importing in React

### ISSUE THREAD

**Issue by sourdragon** [Cainier/gpt-tokens#2]
# My code : 
I am making a function to shorten the number of messages so that the tokens doesnt exceed max tokens.
```jsx
import { GPTTokens } from "gpt-tokens";

let reduceTokens = (history, model = "gpt-3.5-turbo") => {
  // let usageInfo = new GPTTokens({model : model ,messages : history});
  
  // console.log(usageInfo.usedTokens);
  return history;
};

export default reduceTokens; 
```
While importing i am getting this error 

![image](https://user-images.githubusercontent.com/94913289/234484082-a787d177-0538-4f63-9634-a00029801eed.png)

Please help...

**Comment by Cainier** [comment]
OK, I fix it in v1.0.6

> Originally this package was only developed for the node server  > ,0
> Relevant hints have also been added to the README.md

If use vite, you need do this in vite.config.js,

```javascript
import wasm             from 'vite-plugin-wasm'
import topLevelAwait    from 'vite-plugin-top-level-await'
import { defineConfig } from 'vite'

export default defineConfig({
    plugins: [wasm(), topLevelAwait()],
})
```

**Comment by Cainier** [comment]
If use webpack, you can view this

https://www.npmjs.com/package/@dqbd/tiktoken#compatibility
